apps/acls/const.py

```diff
@@ -8,6 +8,6 @@ class ActionChoices(models.TextChoices):
     review = 'review', _('Review')
     warning = 'warning', _('Warn')
     notice = 'notice', _('Notify')
-    notify_and_warn = 'notify_and_warn', _('Notify and warn')
+    notify_and_warn = 'notify_and_warn', _('Prompt and warn')
     face_verify = 'face_verify', _('Face Verify')
     face_online = 'face_online', _('Face Online')
```

Comment:
The provided text is already concise with no obvious issues or differences to be checked. Please share more context of what you'd like to compare between versions or aspects that need analysis if any, so I can assist better.

I hope this information helps! If you have further clarification on something specific, feel free to ask.